src/pygeoprocessing/geoprocessing.py

```diff
@@ -2973,6 +2978,11 @@ def _transform_point(point):
     minx, maxx = sorted([raw_bounding_box[0], raw_bounding_box[2]])
     miny, maxy = sorted([raw_bounding_box[1], raw_bounding_box[3]])
     transformed_bounding_box = [minx, miny, maxx, maxy]
+    if not all(numpy.isfinite(numpy.array(transformed_bounding_box))):
+        raise ValueError(
+            f'some transformed coordinates are not finite: '
+            f'{transformed_bounding_box}, base bounding box may not fit into '
+            f'target coordinate projection system.')
```

Comment:
Love `f` strings, but not necessary on the first and third line! But if you like them there in general, it doesn't really bother me.

Reply:
Similar comment to what I left on #168. James and I both like the consistent f down the line for aesthetic reasons. If you don't agree with that then lets chat more about it.